src/http_handlers/metrics.hpp

```diff
@@ -41,6 +66,9 @@ struct MetricsResponse {
   double average_degree;
   uint64_t memory_usage;
   uint64_t disk_usage;
+  std::vector<std::tuple<std::string, std::string, uint64_t>> event_counters;
+  std::vector<std::tuple<std::string, std::string, uint64_t>> event_gauges;
+  std::vector<std::tuple<std::string, std::string, uint64_t>> event_histograms;
```

Comment:
Does it make sense that these are `utils::pmr::std::vector` as we should measure memory allocated for these counters, gauges and histograms